src/main/java/teammates/logic/core/FeedbackQuestionsLogic.java

```diff
@@ -144,16 +144,16 @@ private boolean areQuestionNumbersConsistent(List<FeedbackQuestionAttributes> qu
      * Checks if there are any questions for the given session that instructors can view/submit.
      */
     public boolean hasFeedbackQuestionsForInstructors(
-            String feedbackSessionName, String courseId, String userEmail) {
+            FeedbackSessionAttributes fsa, boolean isCreator) {
```

Comment:
One suggestion is perhaps use `doesIncludeCreator` ~~`isCreator`~~ given the context of this method?